templates/data/generic_component_data.yaml

```diff
@@ -424,6 +424,7 @@ templates:
             jobs:
               - jobConfig:
                   name: pull-telemetry-manager-build
+                  always_run: true
                   run_if_changed: "^hack/ci/*/.*$|^pkg/*/.*$|^controllers/*/.*$|^charts/*/.*$|^api/*/.*$|^config.yaml$|^Dockerfile$|^(go.mod|go.sum)$|^*/(.*.go|Makefile|.*.sh)|^PROJECT$|^config/*/.*$"
```

Comment:
Remove `run_if_changed`, it's mutual exclusive with `always_run` directive. Your change in data file is not reflected in prowjob definition because of that.

Reply:
run_if_changed removed